website/src/components/DisplayAuthorandReadTime/index.tsx

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+import { useTranslation } from "react-i18next";
+import useStyles from "./styles";
+import { readingTime } from "../../utils/readingTime";
+import { Avatar, Button, Typography } from "@material-ui/core";
+import { getAvatar } from "../../utils/getAvatar";
+
+interface displayAuthorandReadTimeProps {
+  author: string;
+  readTime: string;
+}
+
+// A component to display blog tag and estimate time to read
+const DisplayAuthorandReadTime: React.FC<displayAuthorandReadTimeProps> = ({
+  author,
+  readTime,
+}) => {
+  const classes = useStyles();
+  const { t } = useTranslation();
+  const displayTagandTimeRequiredToRead = () => {
+    return (
+      <>
+        <div className={classes.wrapperBlock}>
+          <span className={classes.author}>
+            <Avatar
+              alt={author}
+              src={getAvatar(author)}
+              className={classes.small}
+            />
```

Comment:
Giving the author first letter as children will help in cases of broken images
ex:
```
<Avatar  alt={author}  src={getAvatar('John H')}  className={classes.small} > J </Avatar>
```
if the image is not present `J ` will come in as avatar

Reply:
Material UI itself handles this functionality if no image available then display the first letter at the place of the avatar.